### PR TITLE
Decentralize loggerific errorMessages.txt

### PR DESCRIPTION
**Pull request by ngfreiter**
This takes the `errorMessages.txt` file which existed only in `shr-cli` and splits it into multiple files, so that each module has its own `errorMessages.txt` file. Also incorrectly numbered or otherwise broken errors were fixed.

PR 4/7
Affects:
- `shr-cli`
- `shr-text-import`
- `shr-expand`
- `shr-fhir-export`
- `shr-json-schema-export`
- `shr-json-javadoc`
- `shr-es6-export`

### CODE REVIEW

File: lib/export.js [hunk 1]
```diff
@@ -4630,4 +4632,8 @@ class Stack {
   }
 }
 
-module.exports = {exportToFHIR, FHIRExporter, exportIG, setLogger, TARGETS, isTargetBasedOn, MODELS_INFO: mdls.MODELS_INFO};
+function errorFilePath() {
+  return require('path').join(__dirname, '../errorMessages.txt');
```
Comment (cmoesel) [inline lib/export.js]: You can probably guess what I'm going to say, right?
```javascript
  return require('path').join(__dirname, '..', 'errorMessages.txt');
```

File: lib/extensions.js [hunk 2]
```diff
@@ -584,7 +584,7 @@ class ExtensionExporter {
         const [type, value2] = this.getSimpleTypeAndValue(originalTypes.map(t => t.code), def, value);
         if (value2 == null) {
           // 13115, 'Couldn\'t identify appropriate Extension.value[x] value for ${elementId}. Is it an attempt to reference a non-Entry?', 'Check value to ensure it is a primitive, maps to a FHIR datatype, or is an Entry (which allows references)', 'errorNumber'
-          logger.error({ elementId: value.effectiveIdentifier }, '13115');
+          logger.error({ elementId: value.effectiveIdentifier.name }, '13115');
```
Comment (cmoesel) [inline lib/extensions.js]: Probably should be `.fqn` rather than `.name`.

**Review by cmoesel — CHANGES_REQUESTED** [review]
Very minor feedback...

File: lib/export.js (hunk 1 above)
Comment (ngfreiter) [reply lib/export.js]: Fixed this here https://github.com/standardhealth/shr-fhir-export/pull/156/commits/bb4474b7d14b86a69751a11ac5823172460b5abf.

File: lib/extensions.js (hunk 2 above)
Comment (ngfreiter) [reply lib/extensions.js]: Fixed this here https://github.com/standardhealth/shr-fhir-export/pull/156/commits/bb4474b7d14b86a69751a11ac5823172460b5abf.

**Comment by cmoesel** [comment]
Changes all look good.  Holding off on merging pending a decision on whether we want this in master yet.  See: https://github.com/standardhealth/shr-cli/pull/227#pullrequestreview-280987415

**Review by cmoesel — APPROVED** [review]
Looks good.  I may squash or rebase this to clean up commits before merging.

**Pull request merged**